python/integration-tests/function/main.tf

```diff
@@ -0,0 +1,22 @@
+resource "aws_lambda_layer_version" "opentelemetry_python_wrapper" {
```

Comment:
Are we considering merge SDK and Collector into one Lambda layer?

Reply:
I might have missed something but I've always been expecting to keep them separate. They're completely separate concepts, so we would release them separately, like lego blocks. E.g., version of collector layer can match collector version, version of sdk layer can match sdk version. 

If we have a combined layer, do we need an uncombined layer for each language for customers that don't need the collector (e.g., export to Zipkin). That's a lot of layers.

Reply:
make sense